test/persistence-hooks.suite.js

```diff
@@ -575,6 +576,204 @@ module.exports = function(dataSource, should) {
           done();
         });
       });
+
+      describe('persist hook', function() {
+        it('triggers `persist` hook on create', function(done) {
+          TestModel.observe('persist', pushContextAndNext());
+
+          TestModel.create(
+            { id: 'new-id', name: 'a name' },
+            function(err, instance) {
+              if (err) return done(err);
+
+              observedContexts.should.eql(aTestModelCtx({
+                data: { id: 'new-id', name: 'a name' }
+              }));
+
+              done();
+            });
+        });
+
+        it('applies updates from `persist` hook - on create', function(done) {
+          TestModel.observe('persist', pushContextAndNext(function(ctx){
+            ctx.data.extra = 'hook data';
+          }));
+
+          TestModel.create(
+            { id: 'new-id', name: 'a name' },
+            function(err, instance) {
+              if (err) return done(err);
+              observedContexts.data.should.have.property('extra', 'hook data');
```

Comment:
Please check that the created `instance` has the extra data. The fact that `observerdContexts.data.extra` is set is not important IMO and it should be already verified by the previous test